astro.config.mjs

```diff
@@ -163,6 +163,13 @@ export default defineConfig({
                 icon: 'switch.svg',
               },
             },
+            {
```

Comment:
I feel that this hsould be part of Workspace, as the toolbox is available in all Workspaces. It enables developer to interact with the workspace programmatically and should also refer to the section in the API for more details. 

Furthermore, i believe  the Headless IDE should be listed under IDEs and they explain how to use it and its functionalities.

ccing @vedranjukic  fro visiblity

Reply:
@stefanicjuraj LSP is part of HIDE but the ask was to refer to the API. I feel the changes do not reflect my comments.

Reply:
```
It utilizes Headless IDE to provide a seamless development experience
```

There is no Headless IDE here.
It would be great if we could have the APIs listed here in some sort of table.

Reply:
I think we will need to carve out (at least in naming the HIDE) as in the GUI view of "daytona list" you should se HIDE instead of VS Code.

Reply:
I have modified the Workspace Toolbox description and added a reference link to the API. Should there be additional table or some other format under the Workspace Toolbox section that will have only the Workspace Toolbox API endpoints? Currently they are all listed in the same API section.

The Headless IDE is added to the IDEs section, but missing a name (should it be specified as HIDE?), and a better description of its capabilities - it should furthermore describe its functionalities and relation to the Workspace Toolbox. Let me know some additional information regarding this. 😄